functions/helloworld/sample_pubsub_test_system.py

```diff
@@ -58,7 +58,7 @@ def test_print_name(publisher_client):
         start_time
     ], stdout=subprocess.PIPE)
     logs = str(log_process.communicate()[0])
-    print logs
-    print start_time
+    print(logs)
```

Comment:
Are these print statements necessary to leave in, or were they only added for development purposes?

Reply:
I will remove them 👌